attributes/gce.rb

```diff
@@ -0,0 +1,15 @@
+default['marketplace_image']['gce']['public']['aio']['image_id'] = 'ami-1e2c0a74'
```

Comment:
LOL I hardcoded the centos7 image in the terraform json which is why I didn't catch this. Thanks for that.